conan/sitara_profile

```diff
@@ -1,5 +1,5 @@
 [build_requires]
-linux/4.4.36@wsbu/stable
+linux/4.9.59@wsbu/stable
```

Comment:
How do we propose to handle that multiple targets use this docker image but not all of them use the same kernel?

Reply:
That is already the case. I mean, conan may not yet support it, but we have both s5t and nglsw and they are on different branches.

Reply:
Sorry, I wasn't specific enough. I meant "start supporting multiple platforms on Conan."
Right now, as far as I know, no one but me and Lance are using Conan, so there's no need for anything other than s5t.

Reply:
Okedoke.